AllReadyApp/Web-App/AllReady.UnitTest/Features/Manage/SendAccountSecurityTokenSmsHandlerTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using AllReady.Features.Manage;
+using AllReady.Services;
+using Moq;
+using Xunit;
+
+namespace AllReady.UnitTest.Features.Manage
+{
+    public class SendAccountSecurityTokenSmsHandlerTests
+    {
+        [Fact]
+        public async Task SendAccountSecurityTokenSmsHandlerInvokesSendSmsAsyncWithCorrectPhoneNumberAndMessage()
+        {
+            var message = new SendAccountSecurityTokenSms { PhoneNumber = "phoneNumber", Token = "code" };
+            var smsMessage = $"Your allReady account security code is: {message.Token}";
+
+            var smsSender = new Mock<ISmsSender>();
+            var sut = new SendAccountSecurityTokenSmsHandler(smsSender.Object);
+            await sut.Handle(message);
+
+            smsSender.Verify(x => x.SendSmsAsync(message.PhoneNumber, smsMessage));
```

Comment:
Not worrying about globalization in this current release and am going to accept with message testing for now